src/v/redpanda/debug_bundle.cc

```diff
@@ -0,0 +1,266 @@
+          "Failed to get 'PATH' environmental variable, the debug bundle may "
+          "be incomplete due to missing dependencies");
+    } else {
+        vlog(bundle_log.trace, "Host path: {}", host_env);
+        host_path = fmt::format("PATH={}", host_env);
+    }
+
+    std::vector<ss::sstring> rpk_argv{
+      _rpk_cmd, "debug", "bundle", "--output", _filename};
+    // Add SASL creds to RPK flags if SASL is enabled.
+    // No need to check for sasl on the broker endpoint because that is for
+    // Kafka brokers where as the bundle is managed by the Admin server only.
+    if (config::shard_local_cfg().enable_sasl()) {
+        rpk_argv.push_back("--user");
+        rpk_argv.push_back(auth_state.get_username());
+        rpk_argv.push_back("--password");
+        rpk_argv.push_back(auth_state.get_password());
+        rpk_argv.push_back("--sasl-mechanism");
+        rpk_argv.push_back(auth_state.get_mechanism());
+    }
+
+    if (params.logs_since.has_value()) {
+        rpk_argv.push_back("--logs-since");
+        rpk_argv.push_back(params.logs_since.value());
+    }
+
+    if (params.logs_until.has_value()) {
+        rpk_argv.push_back("--logs-until");
+        rpk_argv.push_back(params.logs_until.value());
+    }
+
+    if (params.logs_size_limit.has_value()) {
+        rpk_argv.push_back("--logs-size-limit");
+        rpk_argv.push_back(params.logs_size_limit.value());
+    }
+
+    if (params.metrics_interval.has_value()) {
+        rpk_argv.push_back("--metrics-interval");
+        rpk_argv.push_back(params.metrics_interval.value());
+    }
+
+    vlog(
+      bundle_log.trace, "RPK absolute path {}, flags {}", _rpk_cmd, rpk_argv);
+    ssx::background
+      = ss::experimental::spawn_process(
+          _rpk_cmd.c_str(), {.argv = std::move(rpk_argv), .env = {host_path}})
+          .then([this](auto process) {
+              auto stdout = process.stdout();
+              _status = debug_bundle_status::running;
+              return ss::do_with(
+                std::move(process),
+                std::move(stdout),
+                [this](auto& p, auto& stdout) {
+                    return stdout.consume(rpk_consumer(_status)).finally([&p] {
+                        return p.wait().then(
+                          [](ss::experimental::process::wait_status wstatus) {
+                              auto* exit_status = std::get_if<
+                                ss::experimental::process::wait_exited>(
+                                &wstatus);
+                              if (exit_status == nullptr) {
+                                  vlog(
+                                    bundle_log.warn,
+                                    "Failed to run RPK, exit status is "
+                                    "undefined");
+                              }
+
+                              if (exit_status->exit_code != 0) {
+                                  vlog(
+                                    bundle_log.error,
```

Comment:
Should the log on line 187 be error as well or should this be warn?  I think they should be the same level